src/components/TasksTable/index.jsx

```diff
@@ -9,25 +9,33 @@ import ListItem from '@material-ui/core/ListItem';
 import ListItemText from '@material-ui/core/ListItemText';
 import Button from '@material-ui/core/Button';
 import { withStyles } from '@material-ui/core/styles';
-import Typography from '@material-ui/core/Typography';
-import Toolbar from '@material-ui/core/Toolbar';
-import FilterVariantIcon from 'mdi-react/FilterVariantIcon';
+import Hidden from '@material-ui/core/Hidden';
 import LinkIcon from 'mdi-react/LinkIcon';
 import InformationVariantIcon from 'mdi-react/InformationVariantIcon';
 import CloseIcon from 'mdi-react/CloseIcon';
 import { withRouter } from 'react-router-dom';
 import { arrayOf, object } from 'prop-types';
 import { camelCase } from 'change-case';
+import Divider from '@material-ui/core/Divider';
+import TableLargeIcon from 'mdi-react/TableLargeIcon';
+import TableColumnIcon from 'mdi-react/TableColumnIcon';
+import AccountIcon from 'mdi-react/AccountIcon';
 import { formatDistance, differenceInCalendarDays } from 'date-fns';
-import { memoizeWith, omit, pipe, sort as rSort, map } from 'ramda';
+import ArrowDownIcon from 'mdi-react/ArrowDownIcon';
+import ArrowUpIcon from 'mdi-react/ArrowUpIcon';
 import MenuItem from '@material-ui/core/MenuItem';
 import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import Toolbar from '@material-ui/core/Toolbar';
+import DotsHorizontalIcon from 'mdi-react/DotsHorizontalIcon';
```

Comment:
What the "..." icon mean? The more I think about it the more I think we should always show the filtering content area. Having to click multiple buttons (a click for switching the layout and another for open the filtering content area, etc.) can be cumbersome.

I think we should remove this icon and show the filter content area by default.